docs/known-issues.md

```diff
@@ -22,5 +22,5 @@ Known issue = any major user-facing bug or lack of user-facing feature that:
 | Deleting entities directly from the Inspector causes undefined behavior. | 2018-07-17 | [UNR-425](https://improbableio.atlassian.net/projects/UNR/issues/UNR-425) | | No |
 | Having generated files in the symlinked folder `Source/SpatialGDK` means you can’t work in two projects at the same time using the same clone of the SpatialOS Unreal GDK. | 2018-07-23 | [UNR-458](https://improbableio.atlassian.net/browse/UNR-458) | Clone the Unreal GDK separately for each project. | No |
 | We don't support listen servers. | 2018-07-30 | | Use dedicated servers instead. | No |
-| Attempting to use stably-named replicated objects (ie. placed level actors marked for replication) will result in runtime assertions. | 2018-07-31 | [UNR-397](https://improbableio.atlassian.net/browse/UNR-397) | For now, please spawn Actor at runtime instead. | No
 | GameState is sometimes not created/updating properly. In a new deployment this can cause characters to be in a T-pose. | 2018-07-31 | [UNR-475](https://improbableio.atlassian.net/browse/UNR-475) | Try reconnecting until it works. | No
```

Comment:
Drive-by: this is resolved with bridge v2 right? @Vatyx

Reply:
This is fixed by this PR.

Reply:
I think Wanqi means the ```GameState is sometimes not created/updating properly. In a new deployment this can cause characters to be in a T-pose. | 2018-07-31 | [UNR-475](https://improbableio.atlassian.net/browse/UNR-475)```

Reply:
Ah whoops, yes. Sorry was looking at the section in red. We can delete that.